proto/src/determined/api/v1/api.proto

```diff
@@ -1728,6 +1728,16 @@ service Determined {
       tags: "Projects"
     };
   }
+  // Get a list of columns for experiment list table.
+  rpc GetProjectColumns(GetProjectColumnsRequest)
+      returns (GetProjectColumnsResponse) {
+    option (google.api.http) = {
+      get: "/api/v1/projects/{id}/columns"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "Projects"
```

Comment:
should we call this an internal API to give us more flexibility? or do we see it as something users might externally use?

Reply:
I think this should be an internal API for now